test/unit/Reflection/ReflectionClassTest.php

```diff
@@ -995,6 +995,16 @@ public function testGetInterfaceNames() : void
         );
     }
 
+    public function testGetInterfaceNamesFromCustomExceptionClassThatExtendsNativeExceptionClass() : void
+    {
+        $reflector = new ClassReflector(new SingleFileSourceLocator(
```

Comment:
Hmm, it is indeed correct that a single file locator does not know about `Exception`. Does the error occur also with the source locator coming from `BetterReflection\BetterReflection`?

Reply:
Oh yes, seen this just after I commented here: https://github.com/Roave/BetterReflection/issues/456#issuecomment-455983368